version/version.go

```diff
@@ -0,0 +1,50 @@
+package version
```

Comment:
Is there a reason this has to be a separate (now importable) package, as opposed to a file under `main` package as before?

Reply:
Moved back to `main` as discussed